### ISSUE TITLE
prpl-jabber to prpl-xmpp or other ???-xmpp

### ISSUE THREAD

**Issue by Neustradamus** [matrix-org/matrix-bifrost#71]
Can you change prpl-jabber to prpl-xmpp or other ???-xmpp?

Thanks in advance.

**Comment by Half-Shot** [comment]
No, prpl-jabber refers to the plugin in purple's codebase called prpl-jabber. It's actually quite useful as a reminder of how legacy it is and that I should support the xmpp.js implementation as a client.
